docs/Gemfile

```diff
@@ -0,0 +1,12 @@
+# If you do not have OpenSSL installed, change
+# the following line to use "http://"
+source "https://rubygems.org"
+
+# For faster file watcher updates on Windows:
+gem "wdm", "~> 0.1.0", platforms: %s{mswin mingw}
+
+# Windows does not come with time zone data
+gem "tzinfo-data", platforms: %i{mswin mingw jruby}
```

Comment:
Do we need to support Windows?

Reply:
These are the defaults from gov uk tech docs. happy to remove if we don't need to support Windows

Reply:
I have now removed these